src/app/3d/qgs3dmapcanvaswidget.cpp

```diff
@@ -133,6 +135,16 @@ Qgs3DMapCanvasDockWidget::Qgs3DMapCanvasDockWidget( QWidget *parent )
 
   toolBar->addWidget( mBtnOptions );
 
+  mDockUnDockBtn = new QToolButton;
+  mDockUnDockBtn->setIcon( QgsApplication::getThemeIcon( QStringLiteral( "mDockify.svg" ) ) );
+  mDockUnDockBtn->setToolTip( QStringLiteral( "Dock 3D Map View" ) );
```

Comment:
this should be a translatable string...
```suggestion
  mDockUnDockBtn->setToolTip( tr( "Dock 3D Map View" ) );
```